.github/workflows/ubuntu.yml

```diff
@@ -22,6 +22,10 @@ jobs:
       fail-fast: false
       matrix:
         combo:
+          - otp-version: 'latest'
+            elixir-version: 'latest'
+            rebar3-version: 'latest'
+            os: 'ubuntu-latest'
```

Comment:
This would mean that if e.g. OTP, Elixir or `rebar3` were to fail in `latest` the action's results would be affected (which we don't want). As an example, Erlang/OTP Windows 25.2.5.2 has issues installing (I don't know the origin of those), but that shouldn't affect the action's CI results (which I'm pretty sure the results for `windows.yml` will show). Having CI fail and discussing fixes upstream should not be the action's concern.

Reply:
What is your suggestion for an integration test?

Reply:
There is no need for integration tests at this level. You can use the mock `builds.txt` files you added, but otherwise I wouldn't rely on online elements.

Reply:
⚠️ this change has to be reversed, as we won't be doing "online" integration tests, at this moment. Preferably, but it's on you to decide about your own time and energy, we'd have them in the test suites, using the mock `builds.txt`.

Reply:
That has already been tested; what this was catching was getOTPVersions that might be broken. But as I said in my prior PR, I'm not looking to do any major refactoring here.